upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -27,6 +27,12 @@ The life cycle page includes release definitions, support and upgrade requiremen
 
 Upgrades are manually initiated or automatically scheduled. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
 
+//The following note should be removed when the entire HCP fleet is confirmed multi-arch
+[NOTE]
+====
+If your control plane is not currently multi-architecture enabled, the upgrade process will first migrate the cluster to a multi-architecture image and then apply the version upgrade. Multi-architecture clusters are capable of running both x86-based and Arm-based workloads. Clusters created after 25 July, 2024 are multi-architecture enabled by default. 
```

Comment:
```suggestion
If your control plane is not currently multi-architecture enabled, the upgrade process first migrates the cluster to a multi-architecture image and then applies the version upgrade. Multi-architecture clusters are capable of running both `x86_64` and `arm64` architecture workloads. Clusters created after 25 July, 2024 are multi-architecture enabled by default. 
```

Reply:
- https://redhat-documentation.github.io/supplementary-style-guide/#bit-64-arm
- https://redhat-documentation.github.io/supplementary-style-guide/#x86_64
- https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words
SSG says to write it like this. In lieu of this guidance, and also since IBMSG says to avoid hyphenating things unnecessarily, I would avoid using the "-based" phrasing, and just call them architectures.

Reply:
We derive the "Arm-based" instance type identifier from AWS:
https://aws.amazon.com/about-aws/whats-new/2024/06/aws-codebuild-arm-based-workloads-graviton3/